runtime/centrifuge/src/lib.rs

```diff
@@ -2398,6 +2399,10 @@ impl_runtime_apis! {
 		) -> Result<Balance, DispatchError> {
 			Ok(runtime_common::update_nav_with_input(pool_id, input_prices)?.nav_aum)
 		}
+
+		fn cashflow(pool_id: PoolId, loan_id: LoanId) -> Result<Vec<CashflowPayment<Balance>>, DispatchError> {
```

Comment:
nit: maybe better calling this `expected_cashflows`? Since just cashflow implies also including historic cashflows, which is not the case